alerter/alerter.cs

```diff
@@ -1,29 +1,45 @@
 ﻿using System;
+using System.Diagnostics;
 
-namespace AlerterSpace {
-    class Alerter {
+namespace AlerterSpace 
+{
+    class Alerter 
+    {
         static int alertFailureCount = 0;
-        static int networkAlertStub(float celcius) {
-            Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+        static int networkAlertStub(float celcius) 

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.